app/validation/validators.py

```diff
@@ -235,3 +232,39 @@ def __call__(self, form, from_field, to_field):
 
         if from_date == to_date or from_date > to_date:
             raise validators.ValidationError(self.messages['INVALID_DATE_RANGE'])
+
+
+class SumCheck(object):
+    def __init__(self, messages=None, currency=None):
+        if not messages:
+            messages = error_messages
+        self.messages = messages
+        self.currency = currency
+
+    def __call__(self, form, conditions, total, target_total, label):
+        if len(conditions) > 1:
```

Comment:
As discussed this doesn't allow for easy addition of new conditions as your if statement is assuming only a combination or equals or less than.